dist/s_docs

```diff
@@ -90,6 +90,20 @@ structurechk()
 	}
 }
 
+asciichk()
+{
+    # Make sure there are only ASCII characters in the input files.
+    # Odd characters can cause our Python filter to fail.
+    (cd ../src/docs && LC_ALL=C grep -n '[^[:print:]	]' *.dox | cat -v) > $t
+    test -s $t && {
+	echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+	echo 'Non-ASCII content in src/docs/*.dox files is not allowed.'
+	sed -e 's/^/	/' < $t
+	echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+	e=1
+    }
+}
+
```

Comment:
Shouldn't we run this test on every file in the repository? I don't think non-ASCII characters make sense in any file.

Reply:
Thanks, @keithbostic.  I made a ticket for that: WT-7195.